public/app/features/explore/ExplorePage.tsx

```diff
@@ -38,13 +36,9 @@ export default function ExplorePage(props: GrafanaRouteComponentProps<{}, Explor
   // if we were to update the URL on state change, the title would not match the URL.
   // Ultimately the URL is the single source of truth from which state is derived, the page title is not different
   useExplorePageTitle(props.queryParams);
-  const dispatch = useDispatch();
   const { keybindings, chrome } = useGrafana();
   const navModel = useNavModel('explore');
-  const [rightPaneWidthRatio, setRightPaneWidthRatio] = useState(0.5);
-  const { width: windowWidth } = useWindowSize();
-  const minWidth = 200;
-  const exploreState = useSelector((state) => state.explore);
+  const { updateSplitSize, widthCalc, minWidth } = useSplitSizeUpdater(200);
```

Comment:
we don't need to return `minWidth` if it's a parameter, it'd be quite confusing imho.
i think it's better to define it somewhere else (even as a constant outside of the component as `const MIN_PANE_WIDTH=200` or similar) and just pass that in the hook and use the same constant in the warpper

```suggestion
  const { updateSplitSize, widthCalc } = useSplitSizeUpdater(minWidth);
```

Reply:
That makes sense. I've fixed this. I thought this would be a way to keep it sync but you're right it is confusing.